core/src/main/java/fr/sncf/osrd/cli/StandaloneSimulationCommand.java

```diff
@@ -0,0 +1,162 @@
+            required = true
+    )
+    private String infraFilePath;
+    @Parameter(
+            names = {"--sim_path" },
+            description = "Path to the sim railjson file to load",
+            required = true
+    )
+    private String simFilePath;
+    @Parameter(
+            names = {"--res_path" },
+            description = "Path to the result file to save",
+            required = true
+    )
+    private String resultFilePath;
+
+    @Override
+    public int run() {
+
+        try {
+            // Parse the entry files from json
+            var rjsInfra = RJSParser.parseRailJSONFromFile(infraFilePath);
+            var warningRecorder = new WarningRecorderImpl(true);
+            final var signalingInfra = SignalingInfraBuilder.fromRJSInfra(
+                    rjsInfra,
+                    Set.of(new BAL3(warningRecorder)),
+                    warningRecorder);
+            warningRecorder.report();
+
+            var simFile = Files.readString(Path.of(simFilePath));
+            var sim = RJSSimulation.adapter.fromJson(simFile);
+
+            // Parse rolling stocks
+            var rollingStocks = new HashMap<String, RollingStock>();
+            assert sim != null;
+            for (var rjsRollingStock : sim.rollingStocks) {
+                rjsRollingStock.loadingGaugeType = RJSLoadingGaugeType.G1;
+                rollingStocks.put(rjsRollingStock.id, RJSRollingStockParser.parse(rjsRollingStock));
+            }
+
+            // Extract the data to convert RJSTrainSchedule to RJSStandaloneTrainSchedule
+            var standScheduleList = new ArrayList<RJSStandaloneTrainSchedule>();
+
+            for (RJSTrainSchedule sched : sim.trainSchedules) {
+                var standSched = new RJSStandaloneTrainSchedule(
+                        sched.id, sched.rollingStock, sched.initialSpeed, null, sched.stops
+                );
+                standScheduleList.add(standSched);
+            }
+
+            // Parse all train routes
+            var it = rjsInfra.routes.iterator();
+            var allRoutes = new ArrayList<RJSTrainPath.RJSRoutePath>();
+
+            while (it.hasNext()) {
+                var el = it.next();
+                var tracksRoute = new ArrayList<RJSTrainPath.RJSDirectionalTrackRange>();
+
+                for (int i = 0; i < el.path.size(); i++) {
+                    var pathEl = el.path.get(i);
+                    tracksRoute.add(new RJSTrainPath.RJSDirectionalTrackRange(
+                            pathEl.track.id.id, pathEl.begin, pathEl.end, pathEl.direction
+                    ));
+                }
+
+                allRoutes.add(new RJSTrainPath.RJSRoutePath(el.id, tracksRoute, "BAL3"));
+            }
+
+            // Parse the list of train paths and get the results for each one
+            var rjsTrainPathList = new ArrayList<RJSTrainPath>();
+            StringBuilder finalResult = new StringBuilder("{ \"results\": [ ");
```

Comment:
You shouldn't build a json manualy. We're using a library named `Moshi` to do this work for you.